cmd/images-back/main.go

```diff
@@ -79,16 +82,20 @@ func main() {
 }
 func (env *Env) uploadHandler() echo.HandlerFunc {
 	return func(c echo.Context) error {
-		imgTitle := c.FormValue("title") //name
-		ID, err := models.InsertImage(env.db, imgTitle)
+		file, err := c.FormFile("file")
 		if err != nil {
-			log.Fatal(err)
+			log.Print(err)
 		}
 
-		file, err := c.FormFile("file")
+		if getFileContentType(file) != "image" {
+			return echo.NewHTTPError(http.StatusBadRequest, "Please provide valid type of file (image)")
+		}
+		imgTitle := c.FormValue("title") //name
+		ID, err := models.InsertImage(env.db, imgTitle, file.Filename)
 		if err != nil {
-			log.Print(err)
+			log.Fatal(err)
```

Comment:
It is better to use `log.Print()` here and return something to the client. Some error.